### ISSUE TITLE
Grundlagenfrage: Was ist das Standardtemplate für einen Einzelanwalt?

### ISSUE THREAD

**Issue by PBaumfalk** [jlawyerorg/j-lawyer-templates#12]
Wir arbeiten mittlerweile an Falldatenblättern und vorformulierten Schreiben/Schriftsätzen für STANDARDFÄLLE. Wollen wir uns auf eine grundlegende Vorlage wie in den Standardanwaltsprogrammen einigen? Dann kann ich meine Vorlagen für die Standards hier auch teilen ohne meine Kanzleibesonderheiten in der Briefvorlage zu teilen.

**Comment by j-dimension** [comment]
@PBaumfalk wie meinst Du das genau - so in der Art "alle **mitgelieferten** Vorlagen im gleichen Design" o.ä.?

**Comment by PBaumfalk** [comment]
Ja. Jeder kann die so anpassen, wie er möchte, aber J-Lawyer wird mit EINER Standardvorlage ausgeliefert.

**Comment by j-dimension** [comment]
Ja, die bisherigen sind eher auf das Demonstrieren der Funktionalität fokussiert und - was das Design angeht - eher "schlicht". 

Wenn es konkrete Vorschläge / Vorlagen gibt: sehr gern!

**Comment by PBaumfalk** [comment]
Schlicht wäre ja okay. Ich würde mich dann aber bei den Vorlagen auf Word oder auf LibreOffice beschränken? Was wird denn deiner Erfahrung mehr genutzt?

**Comment by PBaumfalk** [comment]
@j-dimension Gibt es denn eine Möglichkeit den Briefbogen dynamisch zu hinterlegen? Also das "Hintergrundbild"? Ich würde dann Werte nach DIN 5008 festlegen für den Standardbriefbogen und darauf die Vorlagen erstellen.

**Comment by j-dimension** [comment]
> Schlicht wäre ja okay. Ich würde mich dann aber bei den Vorlagen auf Word oder auf LibreOffice beschränken? Was wird denn deiner Erfahrung mehr genutzt?

LibreOffice wird aktuell noch mehr genutzt. Mag daran liegen, dass MS Office noch nicht so lange unterstützt wird.

**Comment by j-dimension** [comment]
> Gibt es denn eine Möglichkeit den Briefbogen dynamisch zu hinterlegen? Also das "Hintergrundbild"? Ich würde dann Werte nach DIN 5008 festlegen für den Standardbriefbogen und darauf die Vorlagen erstellen.

Nein, dazu gibt es aktuell nur ein noch offenes Ticket: https://github.com/jlawyerorg/j-lawyer-org/issues/726
